docs/api/python/contrib/summary.md

```diff
@@ -0,0 +1,258 @@
+# Logging MXNet Data for Visualization in TensorBoard
+
+## Overview
+
+The module `mxnet.contrib.summary` enables MXNet users to visualize data in
+[TensorBoard](https://www.tensorflow.org/programmers_guide/summaries_and_tensorboard). 
+Please note that this module only provides the APIs for data logging. For visualization,
+users still need to install TensorBoard.
+
+### How to install TensorBoard
+To launch TensorBoard for visualization, make sure you have the
+[official release of TensorBoard](https://pypi.python.org/pypi/tensorboard) installed.
+You can type `pip install tensorboard` on you machine to install TensorBoard.
```

Comment:
Small comment: the official Tensorboard package requires Tensorflow, but does not declare this dependency in the pypi package (https://pypi.python.org/pypi/tensorboard). So you might need to add a 'pip install tensorflow' here.

Reply:
You are right. Thanks for such a careful and thorough proofreading on the documentation. I will address all the comments as suggested. :)

Reply:
Do we have to make user install tf? how about zihao's extracted tensorboard?

Reply:
If we use it in AMI, then TF is already installed and everything is fine.